Misc/NEWS.d/next/Library/2025-11-27-10-49-13.gh-issue-142006.nzJDG5.rst

```diff
@@ -0,0 +1 @@
+Fix :exc:`email.errors.HeaderWriteError` in :mod:`email.policy.default` which created double newlines when folding long headers with list separators.
```

Comment:
That's where you found the issue, but the issue is a bit more generic than that. I think.  How about "which incorrectly added double newlines when folding lines containing unfoldable syntactic elements wider than max_line_len".